packages/python/plotly/plotly/express/_core.py

```diff
@@ -154,8 +156,8 @@ def invert_label(args, column):
         return column
 
 
-def _is_continuous(df, col_name):
-    return df[col_name].dtype.kind in "ifc"
+def _is_continuous(df: nw.DataFrame, col_name: str) -> bool:
+    return df.get_column(col_name).dtype.is_numeric()
```

Comment:
@FBruzzesi I think this is subtly different from the current logic in the case of unsigned integer types -- the current logic returns `False` for unsigned types but this logic returns `True`. I'm not yet sure whether that makes a difference in practice, but I wanted to flag the difference.

Reply:
True, but I would tend to treat uint and int in the same way. Probably `_is_countinuous` is not the best name, but `_is_numeric` could be.

I will try to come up with an example in which they end up with different plots in master and reason from there